stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -1161,7 +1161,9 @@ function(_add_swift_target_library_single target name)
     set(analyze_code_coverage "${SWIFT_ANALYZE_CODE_COVERAGE}")
   endif()
 
-  if (NOT SWIFTLIB_SINGLE_TARGET_LIBRARY)
+  if(SWIFTLIB_SINGLE_TARGET_LIBRARY)
+    set(lto_type "${SWIFT_STDLIB_LLVM_LTO}")
+  else()
     set(lto_type "${SWIFT_TOOLS_ENABLE_LTO}")
```

Comment:
I think that this may be dead code, not sure if you have the time/interest/inclination to check and cleanup.  It was here before though.

Reply:
Honestly, I fail to understand what does "SWIFTLIB_SINGLE_TARGET_LIBRARY" mean and also that it means when it's set or not set. I just noticed in git history that "being set" is used to mean "we're building the stdlib", so that's how I'm using it here too.

It's used in a whole bunch of places, though, so I'm a bit worried about just eliminating it...

Do you have any more knowledge you could share here?

Reply:
Its dead code, its from before CMake supported Swift properly and the custom build system implementation in Swift.  The `SWIFTLIB_SINGLE_TARGET_LIBRARY` simply meant it was building a target library (which is _always_ true under the stdlib directory) under a single target (i.e. it is one of the slices of the final MachO binary).  The prefix here was for the old name of the function.  I think that cleaning this up, which would be appreciated, is not necessarily in scope for the changes you are doing, and would really be a separate commit any way.